main.tftest.hcl

```diff
@@ -7,6 +7,7 @@ run "valid_serverless_minimal_details" {
     vpc_id         = "my-vpc"
     workgroup_arn  = "workgroup_arn"
     s3_bucket_name = "my-bucket"
+    is_serverless  = true
```

Comment:
Do we need to add some validation that `workgroup_arn` should be supplied when is_serverless is true?